aten/src/ATen/native/transformers/cuda/attention.cu

```diff
@@ -818,6 +830,7 @@ std::tuple<Tensor, Tensor, Tensor, Tensor, c10::SymInt, c10::SymInt, Tensor, Ten
                       query/* Tensor q*/,
                       key/* Tensor k*/,
                       value/* Tensor v*/,
+                      attn_bias_ /* std::optional<Tensor> */,
```

Comment:
While we are, the return std::make_tuple should probably call `std::move` on most of the args.